tests/unit_tests/Objects/new_variable/test_parameter.py

```diff
@@ -1,8 +1,13 @@
 import pytest
 from unittest.mock import MagicMock
 import scipp as sc
+import numpy as np
+
+from scipp import UnitError
 
 from easyscience.Objects.new_variable.parameter import Parameter
+from easyscience.Objects.new_variable.descriptor_number import DescriptorNumber
+from easyscience.Objects.new_variable.descriptor_str import DescriptorStr
```

Comment:
`DescriptorStr` seems unused and ruff somehow didn't catch this

Reply:
Ruff is setup to ignore code in `tests`